### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by lucas-koehler**
- Add issue form templates for bug and feature issues
- Fix typos in SECURITY.md
- Add links to Github discussions and professional support to new issue page

Contributed on behalf of STMicroelectronics

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- type: textarea
+  validations:
+    required: true
+  attributes:
+    label: Expected behavior
+    description: A clear and concise description of what you expected to happen.
+- type: input
+  attributes:
+    label: Cluster provider
+    description: The cluster provider (e.g. Minikube, AWS, GKE, Azure, Docker Desktop). Please include your OS when using a local cluster.
```
Comment (jfaltermeier) [inline .github/ISSUE_TEMPLATE/bug_report.yml:25]:
```suggestion
    description: The cluster provider (e.g. Minikube, AWS, GKE, Azure, Docker Desktop). Please include your OS when using a local cluster. If you are using Terraform, please inlcude the configuration files if possible.  
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- type: textarea
+  validations:
+    required: true
+  attributes:
+    label: Describe alternatives you've considered
+    description: A clear and concise description of any alternative solutions or features you've considered.
+- type: input
+  attributes:
+    label: Cluster provider
+    description: The cluster provider (e.g. Minikube, AWS, GKE, Azure, Docker Desktop) the feature is
```
Comment (jfaltermeier) [inline .github/ISSUE_TEMPLATE/feature_request.yml:26]:
```suggestion
    description: The cluster provider (e.g. Minikube, AWS, GKE, Azure, Docker Desktop) the feature is for. Leave out if the feature is not related to (a) specific provider(s).
```

**Review by jfaltermeier — CHANGES_REQUESTED** [review]
Thanks, looks very good already. I have two minor suggestions. 
Feel free to adjust the wording from the suggestions.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (lucas-koehler) [reply .github/ISSUE_TEMPLATE/bug_report.yml:25]: FYI: I edited you suggestion because it contained a type ;)

**Review by jfaltermeier — APPROVED** [review]

**Pull request merged**